AK/FlyString.h

```diff
@@ -54,6 +54,9 @@ class FlyString {
     // FIXME: Remove this once all code has been ported to FlyString
     [[nodiscard]] DeprecatedFlyString to_deprecated_fly_string() const;
 
+    // Compare this FlyString against another string with ASCII caseless matching.
+    [[nodiscard]] bool equals_ignoring_ascii_case(FlyString const&) const;
```

Comment:
Could this could also take a StingView, which is a bit easier to get from DeprecatedStrings and is literal-friendlier

Reply:
🐝🗻

Reply:
Sure, there are various overloads that could be added. I only needed this one, so I only added this one. :)